client/src/styles/bootstrap_ext/_button.scss

```
@@ -103,11 +103,18 @@ $borderRadius: 1000px;
   }
 }
 
+.btn-secondary {
+  -#{$prefix}btn-active-bg: #{$rk-green-highlight};
```

Invalid CSS var

```suggestion
  --#{$prefix}btn-active-bg: #{$rk-green-highlight};
```

Please check all the CSS, SCSS files.